api/src/pcapi/core/providers/api.py

```diff
@@ -193,6 +192,47 @@ def connect_venue_to_provider(
     return venue_provider
 
 
+def connect_venue_to_allocine(
+    venue: offerers_models.Venue,
+    provider_id: int,
+    payload: providers_models.VenueProviderCreationPayload,
+) -> providers_models.AllocineVenueProvider:
+    if not payload.price:
+        raise providers_exceptions.NoPriceSpecified()
+    if payload.isDuo is None:  # see PostVenueProviderBody
+        raise ValueError("`isDuo` is required")
+
+    pivot = providers_repository.get_allocine_pivot(venue)
+    if not pivot:
+        theater = providers_repository.get_allocine_theater(venue)
+        if not theater:
+            raise providers_exceptions.UnknownVenueToAlloCine()
```

Comment:
`UnknownVenueToAllocine` sera plus homogène que `UnknownVenueToAlloCine`, non ?

Reply:
Mais ça ne vient pas de ta PR 😅; donc je ne te force pas :)

Reply:
J'étais d'accord... et puis j'ai vérifié par curiosité et il semble que la graphie officielle soit "AlloCiné" avec une majuscule (cf. [article Wikipedia](https://fr.wikipedia.org/wiki/Allocin%C3%A9) et [site web d'Allociné](https://www.allocine.fr/service/apropos/)... enfin AlloCiné).

~Je vais de ce pas renommer toutes les occurrences dans notre code : Allocine en AlloCine, allocine en allo_cine, etc.~ ;)

Reply:
Il me semblait aussi que c'était AlloCiné, mais j'aime l'homogénéité 🤩